<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>[PI] Crossbow Bug</title>
    <link>https://forum.moparisthebest.com/t/pi-crossbow-bug/545199</link>
    <description>When ever I attack an npc with a crossbow such as rune or an armadyl crossbow, sometimes it doesn&#39;t cause any damage towards the npc. I&#39;m pretty sure it has to do with the bolt special because it never hits and it randomly doesn&#39;t hit like it doesn&#39;t do damage for 1 hit then the next hit it doesn&#39;t, it&#39;s a random pattern. How can i fix this issue?

[img]http://i.gyazo.com/6a84f417710e69bca8bbe0b5d80d9373.gif[/img]

Error I get: [code][7/19/14 8:46 PM]: java.lang.ArrayIndexOutOfBoundsException: 332
[7/19/14 8:46 PM]: 	at server.model.players.combat.range.RangeExtras.crossbowSpecial(RangeExtras.java:48)
[7/19/14 8:46 PM]: 	at server.model.players.CombatAssistant.crossbowSpecial(CombatAssistant.java:291)
[7/19/14 8:46 PM]: 	at server.model.players.combat.AttackNPC.delayedHit(AttackNPC.java:193)
[7/19/14 8:46 PM]: 	at server.model.players.CombatAssistant.delayedHit(CombatAssistant.java:103)
[7/19/14 8:46 PM]: 	at server.model.players.Client.process(Client.java:675)
[7/19/14 8:46 PM]: 	at server.model.players.PlayerHandler.process(PlayerHandler.java:151)
[7/19/14 8:46 PM]: 	at server.Server$1.execute(Server.java:221)
[7/19/14 8:46 PM]: 	at server.event.Task.tick(Task.java:118)
[7/19/14 8:46 PM]: 	at server.event.TaskScheduler.run(TaskScheduler.java:103)
[7/19/14 8:46 PM]: 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[7/19/14 8:46 PM]: 	at java.lang.Thread.run(Unknown Source)[/code]

If you need the lines provided in the error, I will provide em.</description>
    
    <lastBuildDate>Mon, 21 Jul 2014 04:19:09 +0000</lastBuildDate>
    <category>Runescape</category>
    <atom:link href="https://forum.moparisthebest.com/t/pi-crossbow-bug/545199.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>[PI] Crossbow Bug</title>
        <dc:creator><![CDATA[@joeyscape joeyscape]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/joeyscape">@joeyscape</a> wrote:</p>
          <blockquote>
              <aside class="quote" data-post="2" data-topic="545199">
<div class="title">
<div class="quote-controls"></div>
<img alt width="20" height="20" src="https://forum.moparisthebest.com/letter_avatar/camballer/40/5_e05bb34c421432ee4d40de30c10af3e5.png" class="avatar"> Camballer:</div>
<blockquote>
<p>332 is too large of a value. What is line 48 in RangeExtras?</p>
</blockquote>
</aside>
<pre><code class="lang-auto"></code></pre>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/pi-crossbow-bug/545199/3">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/pi-crossbow-bug/545199/3</link>
        <pubDate>Mon, 21 Jul 2014 04:19:09 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-545199-3</guid>
        <source url="https://forum.moparisthebest.com/t/pi-crossbow-bug/545199.rss">[PI] Crossbow Bug</source>
      </item>
      <item>
        <title>[PI] Crossbow Bug</title>
        <dc:creator><![CDATA[@Camballer Camballer]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/camballer">@Camballer</a> wrote:</p>
          <blockquote>
              <p>332 is too large of a value. What is line 48 in RangeExtras?</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/pi-crossbow-bug/545199/2">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/pi-crossbow-bug/545199/2</link>
        <pubDate>Mon, 21 Jul 2014 02:37:55 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-545199-2</guid>
        <source url="https://forum.moparisthebest.com/t/pi-crossbow-bug/545199.rss">[PI] Crossbow Bug</source>
      </item>
      <item>
        <title>[PI] Crossbow Bug</title>
        <dc:creator><![CDATA[@joeyscape joeyscape]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/joeyscape">@joeyscape</a> wrote:</p>
          <blockquote>
              <p>When ever I attack an npc with a crossbow such as rune or an armadyl crossbow, sometimes it doesn’t cause any damage towards the npc. I’m pretty sure it has to do with the bolt special because it never hits and it randomly doesn’t hit like it doesn’t do damage for 1 hit then the next hit it doesn’t, it’s a random pattern. How can i fix this issue?</p>
<p><img src="http://i.gyazo.com/6a84f417710e69bca8bbe0b5d80d9373.gif" alt width="" height=""></p>
<p>Error I get: <code>[7/19/14 8:46 PM]: java.lang.ArrayIndexOutOfBoundsException: 332
[7/19/14 8:46 PM]: 	at server.model.players.combat.range.RangeExtras.crossbowSpecial(RangeExtras.java:48)
[7/19/14 8:46 PM]: 	at server.model.players.CombatAssistant.crossbowSpecial(CombatAssistant.java:291)
[7/19/14 8:46 PM]: 	at server.model.players.combat.AttackNPC.delayedHit(AttackNPC.java:193)
[7/19/14 8:46 PM]: 	at server.model.players.CombatAssistant.delayedHit(CombatAssistant.java:103)
[7/19/14 8:46 PM]: 	at server.model.players.Client.process(Client.java:675)
[7/19/14 8:46 PM]: 	at server.model.players.PlayerHandler.process(PlayerHandler.java:151)
[7/19/14 8:46 PM]: 	at server.Server$1.execute(Server.java:221)
[7/19/14 8:46 PM]: 	at server.event.Task.tick(Task.java:118)
[7/19/14 8:46 PM]: 	at server.event.TaskScheduler.run(TaskScheduler.java:103)
[7/19/14 8:46 PM]: 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[7/19/14 8:46 PM]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[7/19/14 8:46 PM]: 	at java.lang.Thread.run(Unknown Source)</code></p>
<p>If you need the lines provided in the error, I will provide em.</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/pi-crossbow-bug/545199/1">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/pi-crossbow-bug/545199/1</link>
        <pubDate>Mon, 21 Jul 2014 00:40:05 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-545199-1</guid>
        <source url="https://forum.moparisthebest.com/t/pi-crossbow-bug/545199.rss">[PI] Crossbow Bug</source>
      </item>
  </channel>
</rss>
